modules/accessing-microshift-cluster-remotely.adoc

```diff
@@ -1,48 +1,48 @@
 // Module included in the following assemblies:
 //
-// microshift/microshift-install-rpm.adoc 
+// microshift/microshift-install-rpm.adoc
 
 :_content-type: CONCEPT
 [id="accessing-microshift-cluster-remotely_{context}"]
 = Accessing the MicroShift cluster remotely
 
-Use the following procedure to access the MicroShift cluster from a remote workstation. 
+Use the following procedure to access the {product-title} cluster from a remote workstation.
 
-.Prerequisites 
+.Prerequisites
 
-* You have installed the `oc` CLI. 
+* You have installed the `oc` CLI.
 
-.Procedure 
+.Procedure
 
 [NOTE]
 ====
-The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.  
+The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.
```

Comment:
`op-system` -> {op-system}?

Reply:
@fzdarsky Are we saying `user@microshift` is responsible for setting up the **RHEL for Edge** machine or **host** machine?

Reply:
This section (Accessing MicroShift) is actually just reusing the one written for the "Installing MicroShift RPMs", so we need to be careful the text remains valid for both: It's the (RHEL or RHEL for Edge) machine that runs MicroShift.